packages/consul/src/index.d.ts

```diff
@@ -12,8 +12,13 @@ export interface ConsulOptions {
 }
 
 export interface KeyValueOptions {
-  key?: string;
+  key: string;
```

Comment:
are you sure key should not be optional?

Reply:
yes I believe so